tests/.env.test

```
@@ -10,8 +10,11 @@ NEXT_PUBLIC_ADDRESSABLE_TID=0d48165d8b544e029b816f54c3371ae8
 NEXT_PUBLIC_CUSTOM_RPCS={ "59140": ["https://consensys-zkevm-goerli-prealpha.infura.io/v3/faf4bc4ea7344e5da5e56c55de087480"], "1151111081099710": ["https://chaotic-restless-putty.solana-mainnet.quiknode.pro/", "https://dacey-pp61jd-fast-mainnet.helius-rpc.com/"] }
 NEXT_PUBLIC_DKEY=AmE4rjRHH0aLksP_-Nym-DCL_5qOHBYR77m7svbGyHm5
 NEXT_PUBLIC_BACKEND_URL=https://api-develop.jumper.xyz/v1
-NEXT_PUBLIC_STRAPI_URL=https://strapi.jumper.exchange
-NEXT_PUBLIC_STRAPI_API_TOKEN=636f01bad49bd75c83a32425d2d60ccdbd475cff38ca984e11c3ce9835c79d33f6878d56d251efa6fc2daad4115b6526835976282b22bf3e84a09969e9106b98d90635e0b9a01d26849108dace82817264ff21f448d9b6ee265b0884f21393ea65a67cfed57011aea7f0389d1a547724ff5482101305188c720abe7fcb1c0587
+
+NEXT_PUBLIC_STRAPI_URL=https://strapi-staging.jumper.exchange
+NEXT_PUBLIC_STRAPI_API_TOKEN=19f02160cc3a7592fa401238600e11fbaecfaab515d364ce480d4ea5668e93d3c537c7f01a40dba0b59073ff4478c3bd5cd70546706df1e7f03075b79225f204a597859ca04014fb34fe0f45e96e7ff99eff9dc1ec5a73fe217e50e961688893a6a5881a0643e456f3c606b7c78449998d3f669dfe1c6257e4b83d367ee55a49
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_STRAPI_API_TOKEN key should go before the NEXT_PUBLIC_STRAPI_URL key

(UnorderedKey)